docs/getting-started/first-source.md

```diff
@@ -18,26 +18,50 @@ Create the CloudEvents Player Service:
     --image ruromero/cloudevents-player:latest \
     --env BROKER_URL=http://broker-ingress.knative-eventing.svc.cluster.local/default/example-broker
     ```
+    ==**Expected Output**==
+    ```{ .bash .no-copy }
+    Service 'cloudevents-player' created to latest revision 'cloudevents-player-vwybw-1' is available at URL:
+    http://cloudevents-player.default.127.0.0.1.nip.io
+    ```
+
+    ??? question "Wait, my Revision is named something different!"
+        Since we didn't assign a `revision-name`, Knative Serving automatically created one for us, it's ok if your Revision is named something different.
```

Comment:
Because we didn't assign a `revision-name`, Knative Serving automatically created one for us. It's okay if your revision is named something different.